docs/docs/target/container_image.md

```diff
@@ -375,10 +375,39 @@ $ skopeo copy docker-daemon:alpine:3.11 oci:/path/to/alpine
 $ trivy image --input /path/to/alpine
 ```
 
-## SBOM generation
+## SBOM
+Trivy supports the generation of Software Bill of Materials (SBOM) for container images and the search for SBOMs during vulnerability scanning.
+
+### Generation
 Trivy can generate SBOM for container images.
 See [here](../sbom/index.md) for the detail.
 
+### Discovery
+Trivy can search for Software Bill of Materials (SBOMs) that reference container images.
```

Comment:
maybe specify in this sentence where the 'search' happens?

Reply:
>maybe specify in this sentence where the 'search' happens?

This page is for container images (`docs/docs/target/container_image.md`). Do you mean something else?

> then why would we want trivy to be able to discover the SBOMs related to that image

Added https://github.com/aquasecurity/trivy/pull/3768/commits/537dcea300ed0dfbc6f1ff53f8f9bc869c8a5ab7. Thanks.